src/events/writers-competition-2024q4/overview.mdx

```diff
@@ -12,7 +12,7 @@ Web3 developers of all levels—beginner, intermediate, and advanced—who are p
 1. **Pick a topic** from the Prizes list down below. Each topic includes detailed expectations and a corresponding bounty for the article.
 2. **Choose up to three topics** from the board. Every participant can submit up to three articles of their choice, only one winner will be selected per topic.
 3. **Optional**: Share what you're working on via our [Telegram group](https://t.me/+0tvdw8QMJBMyOTli).
-4. **Submit your article** by the deadline 25th Nov 2024, 11:59 (UTC). It must be published on a public blog (e.g., Mirror, Hashnode, Hackmd, dev.to, Devpost, Paragraph, Medium, etc.) under an open-source license such as MIT, GPL, or CC BY.
+4. **Submit your article** by the deadline Dec 9th 2024, 11:59 (UTC). It must be published on a public blog (e.g., Mirror, Hashnode, Hackmd, dev.to, Devpost, Paragraph, Medium, etc.) under an open-source license such as MIT, GPL, or CC BY.
 5. **Winners** will be contacted via Telegram (ensure your contact information is up to date).
 6. **Stick to testnets (or devnets):** Since the tutorials are for developers, please use testnets to ensure no real money is required.
 7. **Submissions** must be in English and must follow this format:
```

Comment:
Revise blurb to include preference of English but accept submission in other languages